client/src/constants/payment_server.js

```diff
@@ -0,0 +1,5 @@
+const PAYMENT_SERVER_URL = process.env.NODE_ENV === 'production'
```

Comment:
This is very simple but why do we need to state production here? Since I am new to Stripe as well. I understand it is loking to see if it in development or in production. What makes the difference?